pkg/attestation/crafter/runners/gitlabpipeline.go

```diff
@@ -16,15 +16,30 @@
 package runners
 
 import (
+	"context"
 	"os"
 
 	schemaapi "github.com/chainloop-dev/chainloop/app/controlplane/api/workflowcontract/v1"
+	"github.com/chainloop-dev/chainloop/pkg/attestation/crafter/runners/oidc"
+	"github.com/rs/zerolog"
 )
 
-type GitlabPipeline struct{}
+type GitlabPipeline struct {
+	gitlabToken *oidc.GitlabToken
+}
+
+func NewGitlabPipeline(ctx context.Context, logger *zerolog.Logger) *GitlabPipeline {
+	client, err := oidc.NewGitlabClient(ctx, logger)
+	if err != nil {
+		logger.Debug().Err(err).Msg("failed to create Gitlab OIDC client")
```

Comment:
Please wrap the former error `Msgf("failed to create Gitlab OIDC client: %w", err)`

Reply:
Done.